### ISSUE TITLE
install

### ISSUE THREAD

**Issue by dcrystalj** [newhoa/victory-icon-theme#1]
Can you provide installing procedure for this icon set?

**Comment by AdrianKoshka** [comment]
It seems (from my limited poking around) you can just `clone` it and `cp` it to `~/.icons` if you wanted (or `/usr/share/icons`), Though as long as I'm correct and @newhoa approved I wouldn't mind writing a simple install guide for the README.md.

**Comment by dcrystalj** [comment]
i can confirm `sudo git clone https://github.com/newhoa/victory-icon-theme.git /usr/share/icons`
works!

**Comment by AdrianKoshka** [comment]
Yay! Still wouldn't be bad to add at least a little blurb in the README. In my opinion.

**Comment by newhoa** [comment]
Thanks Adrian! Yeah, my fault for not documenting install instructions. But yes, you can clone it.

The clone URL is on the [main theme github page](https://github.com/newhoa/victory-icon-theme) in the top right, it's green and says "Clone or Download" and when you click that button it will provide you the clone url or download option. 

In the terminal you can cd to your icon directory and git clone this repository there.

> cd /home/yourusername/.icons/
> 
> git clone https://github.com/newhoa/victory-icon-theme.git

The command that dcrystalj used will clone it as root to the system icon directory, allowing you to see it in root applications, and allowing all users on the machine to select it.

> sudo git clone https://github.com/newhoa/victory-icon-theme.git /usr/share/icons

After cloning, open your appearance manager to select the theme. The theme/appearance managers I know off the top of my head are:

Gnome 3: gnome-tweak-tool
XFCE: xfce4-appearance-settings
LXDE/LXQT: lxappearance

All should be found in the Settings section of your DE. If anyone knows the names of the MATE, Cinnamon, Budgie, or other DE installer please let me know so I can add it to the instructions when I update.

Alternatively, you can download the theme using the same green button on the main theme page. Then just extract it to /home/yourusername/.icons and use your theme/appearance manager to select the icon theme.

Hopefully all this makes sense. If anyone else needs help please feel free to comment in this thread. Thanks again Adrian for helping out! I'll get these instructions included in the next update!
- Darin

**Comment by AdrianKoshka** [comment]
@newhoa Thanks! I'll fork the repo (not my first rodeo :P ) and try to cobble instructions together from this quick little template you wrote up. Then submit a PR. if you would prefer that, rather then you writing the instructions.

**Comment by newhoa** [comment]
Thanks for the offer, I started adding the instructions to the readme as I typed them up to this reponse, I just haven't finished or updated it yet.

I have no idea how any of this works really, I'm new to github and this type of system. If you already had something done or planned feel free to go for it. I assume what your describing is something I could merge into my repository? That sounds neat. Sorry for my ignorance haha I really am just learning about all this!
- Darin
